core/cautils/getter/gcpcloudapi.go

```diff
@@ -0,0 +1,35 @@
+package getter
+
+import (
+	"context"
+
+	containeranalysis "cloud.google.com/go/containeranalysis/apiv1"
+)
+
+type GCPCloudAPI struct {
+	credentialsPath  string
+	context          context.Context
+	client           *containeranalysis.Client
+	projectID        string
+	credentialsCheck bool
+}
+
+var globalGCPCloudAPIConnector *GCPCloudAPI
+
+func GetGlobalGCPCloudAPIConnector() *GCPCloudAPI {
+	// need to move this to function where creds will be added 
```

Comment:
This looks like an open issue?

Reply:
For credentials I don't know how armo adaptor fetches credentials so cred thing is still open and needs to be managed